Content.Server/SurveillanceCamera/Systems/SurveillanceCameraSystem.cs

```diff
@@ -271,6 +275,8 @@ public void SetActive(EntityUid camera, bool setting, SurveillanceCameraComponen
 
         if (setting)
         {
+            var attemptEv = new SurveillanceCameraSetActiveAttemptEvent();
+            RaiseLocalEvent(camera, ref attemptEv);
             component.Active = setting;
         }
```

Comment:
you arent actually using this event for anything?

Reply:
uh oh